sdk/inc/azure/storage/az_storage_blobs.h

```diff
@@ -76,17 +84,20 @@ AZ_NODISCARD az_result az_storage_blobs_blob_client_init(
     void* credential,
     az_storage_blobs_blob_client_options* options);
 
+/**
+ * @brief Azure Storage Blobs Blob upload options.
+ * @remark Reserved for future use
```

Comment:
What does this mean? This seems like unnecessary info for the end user. I would remove the remark. You can put such a comment on the `unused` field within the `_internal` nested struct as a note as part of the hidden implementation detail.